packages/middleware-flexible-checksums/src/types.ts

```diff
@@ -0,0 +1,22 @@
+import { ChecksumAlgorithm } from "./configuration";
```

Comment:
Nit: rename file to maybe `constants.ts`? Since they are const not types.

Reply:
Renamed in [`56dc06f` (#3340)](https://github.com/aws/aws-sdk-js-v3/pull/3340/commits/56dc06f9907fcb1448c8ee4008cef2f663fa0f5d)